packages/analytics/metric-cards/src/components/display/MetricsCard.vue

```diff
@@ -130,7 +131,7 @@ const cardDisplayFull = [MetricCardSize.Medium, MetricCardSize.Large].includes(p
 
 const metricFontSize = props.cardSize === MetricCardSize.ExtraLarge
   ? '48px'
```

Comment:
We have `kui-font-size-100` is 48px, if you wanna use it here

Reply:
Done